src/index.ts

```diff
@@ -114,14 +113,21 @@ export default class ShadingScene {
    * This function is called as a last step, after the scene is fully build.
    * It runs the shading simulation and returns a THREE.js colored mesh.
    * The colors are chosen from the viridis colormap.
-   * @param numberSimulations Number of random sun positions that are used to calculate the PV yield
-   * @param diffuseIrradianceURL URL where the files for the diffuse Irradiance can be retreived
-   * @returns
+   * @param numberSimulations Number of random sun positions that are used to calculate the PV yield.
+   * @param diffuseIrradianceURL URL where the files for the diffuse Irradiance can be retreived.
+   * @param pvCellEfficiency Efficiency of the solar cell, usually this is a value close to 0.2.
+   * @param maxYieldPerSquareMeter Upper boundary of the mesh color in kWh/m2/year.
+   * In Germany this is something like 1400 kWh/m2/year multiplied with the given pvCellEfficiency.
+   * @param progressCallback function that handles the progress of the simulation, used for showing a
+   * loading bar on a website
+   * @returns A three.js colored mesh of the simulationGeometry.
    */
 
   async calculate(
     numberSimulations: number = 80,
     diffuseIrradianceURL: string | undefined,
+    pvCellEfficiency: number,
+    maxYieldPerSquareMeter: number,
```

Comment:
`diffuseIrradianceURL` is optional, but `pvCellEfficiency` and `maxYieldPerSquareMeter` are not. Would it make sense to move the optional argument to the back of the argument list? Or we could switch to a dict of arguments, i.e.

```
scene.calculate(80, {
  diffuseIrradianceURL: 'https://...',
  progressCallback: function(i) {},
})
```

Reply:
Yes, this dict option seems good.